resources/view/modules/analytics/google.html.twig

```diff
@@ -0,0 +1,13 @@
+<script type="text/javascript">
```

Comment:
Please can you rename this file to google-analytics.html.twig to match the config file?